cost-analyzer/values.yaml

```diff
@@ -202,6 +202,50 @@ kubecost:
     #  cpu: "100m"
     #  memory: "256Mi"
 
+# Kubecost Metrics deploys a separate pod which will emit kubernetes specific metrics required 
```

Comment:
aren't we deploying in our current container first? if so, should we wait on values updates?

Reply:
This is disabled by default, and since the internal version requires updates to env vars, we need to at least template the values.